tests/integration/test_packages.py

```diff
@@ -33,6 +35,9 @@ def test_packages(env_package, env_name, test_env, tmpdir):
     * Check that the source tarball includes expected deps directory
     * Check: The content manifest is successfully generated and contains correct content
     """
+    if env_name == "missing_gomod" and "cachito-prod" not in str(os.environ.get("JOB_NAME")):
+        pytest.skip("Test is skipped in local environment")
```

Comment:
I think Athos prefers to change the config instead, which sounds reasonable. Instead of skipping the test, you can set `cachito_gomod_ignore_missing_gomod_file = True` in the `DevelopmentConfig`

Reply:
ok, I misunderstood what Athos meant

Reply:
changed that